### ISSUE TITLE
7.1.0 breaks docker - peer dependency module 'helmet'

### ISSUE THREAD

**Issue by litoj** [venables/koa-helmet#92]
I have spent about 10 hours trying to figure this out and this is what I know for sure:

- 7.0.2 worked both locally as well as when built and ran through docker
- 7.1.0 works locally, not when built through docker unless using webpack to include the source code in a singular file

By broken I mean that my app cannot find this module under `node_modules` when installed through docker, but can when installed locally:
```
Cannot find module 'helmet'
Require stack:
- /app/node_modules/koa-helmet/lib/koa-helmet.js
- /app/build/app.js
```

I am sorry to not have anything more specific, but this is all I've managed to gather.

**Comment by joekrill** [comment]
I just came across this problem, but I'm using Yarn PnP so the error was a little more helpful:

> koa-helmet tried to access helmet (a peer dependency) but it isn't provided by your application; this makes the require call ambiguous and unsound.

So if you installing `helmet` as a dependency in your app should fix things for you.

It looks like [this commit](https://github.com/venables/koa-helmet/commit/bf76e0143d2fe9708418881f0febd79e7335b5bd) changed `helmet` from a dependency to a peerDependency. 

I think probably:

1. v7.1.0 should probably have a been a major version bump -- this seems like a breaking change to me? 
2. The README should reflect the change in the installation instructions. 

I've never contributed here before but I can certainly provide a PR for the 2nd issue. I don't know how releases are handled but maybe it should be considered to do a v7.1.1 release that reverts back to using helmet as a dependency, then release a version with this change as v8.0?

**Comment by venables** [comment]
Thanks @litoj, @joekrill , and @kdevan  . And I apologize for the breaking change in 7.1.0 -- it should have been a major release.  

The latest fixes should be available in v8.0.0.

Please let me know if you have other issues with this.  I'll leave the issue open to make sure.
